server/php-engine.cpp

```diff
@@ -896,7 +890,7 @@ static void send_rpc_error(connection *c, long long req_id, int error_code, cons
 }
 
 int rpcx_execute(connection *c, int op, raw_message *raw) {
-  vkprintf(1, "rpcx_execute: fd=%d, op=%d, len=%d\n", c->fd, op, raw->total_bytes);
+  vkprintf(1, "rpc execute: fd=%d, op=%d, len=%d\n", c->fd, op, raw->total_bytes);
```

Comment:
it's called when any engine response comes, so on production it will be called many many times. Maybe it's better to increase it upto 2

Reply:
apply it